lens-labels/src/Lens/Labels/Function.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+------------------------------------------------------------------------------
+-- | This module provides OverloadedLabels 'IsLabel' support to lenses
+-- generated by protolens; eg, it allows you to reference a proto field 'foo'
+-- as @#foo@ without having protolens steal perfectly good identifier names.
```

Comment:
Nit: theoretically this package isn't specific to `proto-lens`, so maybe don't mention protos in the comment (just "records" more generally)?

Reply:
Whoops, just copied your comment across without thinking